src/foremast/awslambda/awslambda.py

```diff
@@ -61,6 +61,22 @@ def _check_lambda(self):
             pass
         return exists
 
+    def _check_lambda_alias(self):
```

Comment:
Thought about that while developing. LambdaFunction class presumes to have prop_path defined for a lot of its functions and handling missing prop_path would introduce more complex logic in handling that. 

We can call external function from `utils/` of course but then it would introduce "dependency" to another function (not that we don't have many already) and class itself won't reuse one API client. Since LambdaFunction will be executed more often (since it does the deploys) i think that having the atomic and "one-api-call" version is a good thing because each API call that we save at large scale will be a good thing.

I am against duplication of code but for me in this case it makes sense to have two similar functions. Of course, if you do like me to change it let me know which approach you would like.